examples/neg/parsing/blockargs.check

```diff
@@ -1,3 +1,6 @@
-[error] examples/neg/parsing/blockargs.effekt:3:20: Effects not allowed here. Maybe you mean to use a function type `() => T / E`?
+[error] examples/neg/parsing/blockargs.effekt:3:18: Expected block type, but got a type-and-effect annotation R / Exc.
 def foo[R] { f : R / Exc }: R = <>
-                   ^
+                 ^^^^^^^
+[info] examples/neg/parsing/blockargs.effekt:3:18: Did you mean to use a function type () => R / Exc?
```

Comment:
Actually, in this case I always use the short hand notation `=> R / Exc` which we could also mention (additionally).

Reply:
I considered that, but the error message got a bit too bloated and a bit harder to explain. If we always use just `() => ...` instead of `=> ...`, it's, in a way, more obvious as to what's going on?

Reply:
Yeah... I don't like `=> ...` in a sense since it breaks the principle of definitionsite=usesite. We allowed `R / Exc` in former times but that was ambiguous to parse. Maybe we can allow it again now???